services/src/test/scala/cromwell/pipeline/service/ProjectConfigurationServiceTest.scala

```diff
@@ -159,5 +171,51 @@ class ProjectConfigurationServiceTest extends AsyncWordSpec with Matchers with M
           .map(_ should have.message("Access denied. You  not owner of the project"))
       }
     }
+    "build configuration" should {
+
+      "return success message for request" taggedAs Service in {
+
+        when(configurationRepository.getAllByProjectId(projectId)).thenReturn(Future.successful(Seq()))
+        when(projectService.getUserProjectById(projectId, userId)).thenReturn(Future.successful(dummyProject))
+        when(projectVersioning.getFile(dummyProject, projectFile.path, optionVersion))
+          .thenReturn(Future.successful(Right(projectFile)))
+        when(womTool.inputsToList(projectFileContent.content)).thenReturn(Right(Nil))
+
+        configurationService
+          .buildConfiguration(projectId, projectFile.path, optionVersion, userId)
+          .map(
+            builtConfiguration =>
+              builtConfiguration shouldBe ProjectConfiguration(
```

Comment:
As for me it's a little bit confusing buildConfiguration / builtConfiguration. Should we use another name? Or just underscore?

Reply:
Thanks, that's indeed confusing. My task was just to move this code. I would prefer not to change it or change as little as possible